distributions/Lakka/options

```diff
@@ -512,3 +512,4 @@ DISTRO_SRC="http://sources.libreelec.tv/$LIBREELEC_VERSION"
       LIBRETRO_CORES+=" mame2000  fbalpha2012 "
     fi
   fi
+LIBRETRO_CORES="${LIBRETRO_CORES// chailove /}"
```

Comment:
I'm afraid this will disable` chailove` for all targets.

Reply:
yes, many target already disable `chailove`

Reply:
tbh am ok with this

Reply:
Me too! :+1:

Reply:
build issue resolved, reenable it.